src/Bridge/Symfony/Bundle/Resources/schema/xml_metadata.xsd

```diff
@@ -0,0 +1,73 @@
+<xs:schema attributeFormDefault="unqualified" elementFormDefault="qualified" xmlns:xs="http://www.w3.org/2001/XMLSchema">
```

Comment:
The XSD isn't related with Symfony (it will be useful if you use the standalone library too).

I suggest `src/schema/metadata.xsd`